pyproject.toml

```diff
@@ -25,22 +25,32 @@ profile = "black"
 [tool.mypy]
 show_error_codes = true
 strict = true
+disable_error_code = [
```

Comment:
What was the reason of disabling the errors related to `import`?

Reply:
No reason, I copied across some code from topo-procesor including this and the mypy overrides below.
I can remove it, and we can add it back as we go if its needed?

Reply:
Let's push it and do the cleaning in another PR